### DISCUSSION TITLE
Do we need an explicit Contributor License Agreement in place?

### DISCUSSION THREAD

**Discussion by pjohnmeyer** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#92] [General]
https://en.wikipedia.org/wiki/Contributor_License_Agreement

Given the limited license we intend to adopt, do we need an explicit CLA?

If so, is the need different for coalition members versus non-member (public) contributions?

**Comment by JSv4** [comment]
Good question. It's been discussed and was tabled for the time being. I can bring it up again with the Steering Committee.

**Comment by JSv4** [comment]
Let's revisit once this initial release is done and we start to bring in more contributors.

**Reply by JSv4** [reply]
@pjohnmeyer, I can't mark this as answered for some reason. If it lets you, can you please do it?

**Reply by pjohnmeyer** [reply]
@JSv4 only "Q&A" can be marked as answered.

I think this an issue we at minimum need a clear "yes" or "no" on before we open the repo up to the public, we don't want to have to scramble to create one. As I hinted, I suspect this is only an issue if individuals that are **_not_** representing consortium members wish to contribute -- which seems pretty unlikely?

**Reply by JSv4** [reply]
Didn't even realize this wasn't a Q&A, lol. The UI looks really similar for questions vs discussions. 

At the moment, yes, very unlikely to take contributions from non-member orgs. The last time we brought this up, it was a definitive "not now, not for the existing team / orgs". I can bring it up again.
